tests/GeneralStateTestsRunner.js

```diff
@@ -77,9 +76,6 @@ function runTestCase (options, testData, t, cb) {
       }
     },
     function (done) {
-      if (options.jsontrace) {
-        console.log('{ "steps": [' + steps + ']}')
```

Comment:
If you print it in place, you need to print `{ "steps":[` here, the comma in place and `]}` at the end.

Reply:
I made the changes to make the output conform to the example here: https://github.com/ethereum/tests/issues/249 .

There are no commas.  Each step is logged on its own line.

Reply:
Is this PR still far away (sorry, didn't really follow discussion)? Would be really useful, I always monkey-path this stuff together again and again...

Reply:
@jwasinger ah, that original issue isn't really clear on a spec.

Reply:
@holgerd77 This is ready to merge.

Reply:
Probably @cdetrio should review this, since he has started the initiative. Casey?